src/runtime_src/xdp/profile/plugin/user/user_plugin.cpp

```diff
@@ -26,34 +26,33 @@ namespace xdp {
   {
     UserEventsPlugin::live = true;
 
-    db->registerPlugin(this) ;
-    db->registerInfo(info::user) ;
+    db->registerPlugin(this);
+    db->registerInfo(info::user);
 
-    VPWriter* writer = new UserEventsTraceWriter("user_events.csv") ;
-    writers.push_back(writer) ;
+    VPWriter* writer = new UserEventsTraceWriter("user_events.csv");
+    writers.push_back(writer);
   }
 
   UserEventsPlugin::~UserEventsPlugin()
   {
-    if (VPDatabase::alive())
-    {
+    if (VPDatabase::alive()) {
       // We were destroyed before the database, so write the writers
       //  and unregister ourselves from the database
       for (auto w : writers) {
-        w->write(false) ;
+        w->write(false);
         db->getStaticInfo().addOpenedFile(w->getcurrentFileName(), "VP_TRACE");
       }
-      db->unregisterPlugin(this) ;
+      db->unregisterPlugin(this);
     }
     UserEventsPlugin::live = false;
   }
 
   void UserEventsPlugin::writeAll(bool openNewFiles)
```

Comment:
XRT code also puts class types on a separate line like:
void UserEventsPlugin::
writeAll(bool openNewFiles)

I'd propose we follow that.